docs/tempo/website/setup/set-up-test-app.md

```diff
@@ -0,0 +1,163 @@
+
+The example agent Kubernetes ConfigMap configuration below opens many trace receivers (note that the remote write is onto the Tempo cluster using OTLP gRPC):
+
+```yaml
+kind: ConfigMap
+metadata:
+  name: grafana-agent-traces
+apiVersion: v1
+data:
+  agent.yaml: |
+    traces:
+        configs:
+          - batch:
+                send_batch_size: 1000
+                timeout: 5s
+            name: default
+            receivers:
+                jaeger:
+                    protocols:
+                        grpc: null
+                        thrift_binary: null
+                        thrift_compact: null
+                        thrift_http: null
+                opencensus: null
+                otlp:
+                    protocols:
+                        grpc: null
+                        http: null
+                zipkin: null
+            remote_write:
+              - endpoint: <tempoDistributorServiceEndpoint>
+                insecure: true  # only add this if TLS is not required
+            scrape_configs:
+              - bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+                job_name: kubernetes-pods
+                kubernetes_sd_configs:
+                  - role: pod
+                relabel_configs:
+                  - action: replace
+                    source_labels:
+                      - __meta_kubernetes_namespace
+                    target_label: namespace
+                  - action: replace
+                    source_labels:
+                      - __meta_kubernetes_pod_name
+                    target_label: pod
+                  - action: replace
+                    source_labels:
+                      - __meta_kubernetes_pod_container_name
+                    target_label: container
+                tls_config:
+                    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                    insecure_skip_verify: false
+```
+
+If you have followed the [Tanka Tempo installation example]({{< relref "../setup/tanka.md" >}}), then the `endpoint` value would be:
+
+```bash
+distributor.tempo.svc.cluster.local:4317
+```
+
+Apply the ConfigMap with:
+
+```bash
+kubectl apply --namespace default -f agent.yaml
+```
+
+Deploy the Grafana Agent using the instructions from one the relevant instructions above.
+
+## Create a Grafana Tempo data source
+
+To allow Grafana to read traces from Tempo, you must create a Tempo data source.
+
+1. Navigate to **Configuration ≫ Data Sources**.
+
+2. Click on **Add data source**.
+
+3. Select **Tempo**.
+
+4. Set the URL to `http://<tempo-host>:<http_listen_port>/`, filling in the path to your gateway and the configured HTTP API prefix. If you have followed the [Tanka Tempo installation example]({{< relref "../setup/tanka.md" >}}), this will be: `http://query-frontend.tempo.svc.cluster.local:3200/`
+
+5. Click **Save & Test**.
```

Comment:
```suggestion
1. Navigate to **Configuration ≫ Data Sources**.

1. Click on **Add data source**.

1. Select **Tempo**.

1. Set the URL to `http://<tempo-host>:<http_listen_port>/`, filling in the path to your gateway and the configured HTTP API prefix. If you have followed the [Tanka Tempo installation example]({{< relref "../setup/tanka.md" >}}), this will be: `http://query-frontend.tempo.svc.cluster.local:3200/`

1. Click **Save & Test**.
```

Reply:
We generally want place holder text to be capitalized so it catches the eye. So like, `http://<TEMPO-HOST>:<HTTP-LISTEN-PORT>` 
I think we also decided on n-dashes rather than underscores for readability

Reply:
Done.